my-app/src/components/Posts/index.jsx

```diff
@@ -11,6 +16,34 @@ export const SmallPost = ({ post }) => {
         ))}
       </div>
       <div>{post.like_users.length > 0 && `❤️ ${post.like_users.length}`}</div>
+    </Link>
+  );
+};
+export const BigPost = ({ post }) => {
+  const onClickLike = () => {
+    alert("좋아요");
+  };
+  return (
+    <div className="py-3 w-full bg-orange-400 text-black border-0 ring-4 ring-orange-200 rounded-xl font-medium">
+      <div className="px-8">
+        <h1 className="font-extrabold text-2xl">
+          {post.author.username}의 {post.title}
+        </h1>
```

Comment:
세미나 자료에서 PostDetailPage 화면 캡처를 보면 포스트가 등록된 시간 정보가 우측 상단에 있는 걸 확인할 수 있는데요, 이 부분도 구현해 주시면 좋을 것 같습니다!
```suggestion
        <span className="text-black font-medium text-base">
          {post.created_at.slice(0, 10)}
        </span>
```